tests/e2e/utils/helpers.go

```diff
@@ -238,3 +255,13 @@ func CheckKeyEquality(keyPath1, keyPath2 string) (bool, error) {
 
 	return string(key1) == string(key2), nil
 }
+
+// Currently downloads subnet-evm, but that suffices to test the custom vm functionality
+func DownloadCustomVMBin() (string, error) {
+	targetDir := os.TempDir()
+	subnetEVMDir, err := binutils.DownloadReleaseVersion(logging.NoLog{}, subnetEVMName, subnetEVMVersion, targetDir)
+	if err != nil {
+		return "", err
+	}
```

Comment:
Nit: As this is a test, I think it might be a good idea to add a check (`os.Stat`) that the file is indeed there.